packages/expo-notifications/android/src/main/java/expo/modules/notifications/notifications/model/RemoteNotificationContent.kt

```diff
@@ -65,21 +59,17 @@ class RemoteNotificationContent(private val remoteMessage: RemoteMessage) : INot
     get() = remoteMessage.notification?.color?.let { android.graphics.Color.parseColor(it) }
 
   // NOTE the following getter functions are here because the local notification content class has them
-  // and this class conforms to the same interface. They are not supported by FCM.
-  override val isAutoDismiss: Boolean
-    get() = remoteMessage.data["autoDismiss"]?.toBoolean() ?: true
+  // and this class conforms to the same interface.
+  // They are not supported by FCM but were previously implemented by JSONNotificationContentBuilder.java.
+  override val isAutoDismiss = notificationData.autoDismiss
 
-  override val categoryId: String?
-    get() = remoteMessage.data["categoryId"]
+  override val categoryId = notificationData.categoryId
 
-  override val isSticky: Boolean
-    get() = remoteMessage.data["sticky"]?.toBoolean() ?: false
+  override val isSticky = notificationData.isSticky
 
-  override val subText: String?
-    get() = remoteMessage.data["subtitle"]
+  override val subText = notificationData.subText
```

Comment:
Typo here and in `NotificationData.kt`, should be `subtitle` instead of `subText`, otherwise build breaks as it doesn't override `subtitle` from `INotificationContent` then.

Reply:
`subtitle` was renamed to `subText` internally in the android package on the main branch (not a change that would influence consumers) so this is correct.